### ISSUE TITLE
Test code is incorrect.

### ISSUE THREAD

**Issue by imgits** [l33tdaniel/mnist-ai-example#1]
Your test code is incorrect. You are using the same training set for testing instead of the test set. The corrected code is as follows:

```
correct = 0
total = 0
with torch.no_grad():
    for data in testloader:
        inputs, labels = data     #Should add this line
        images = inputs.to(device)
        labels = labels.to(device)
        outputs = net(images)
        _, predicted = torch.max(outputs.data, 1)
        total += labels.size(0)
        correct += (predicted == labels).sum().item()

print(f'Accuracy of the network on the 10000 test images: {100 * correct / total} %')
```

**Comment by l33tdaniel** [comment]
Thank you! Just updated it
